Userland/Applications/Browser/Tab.cpp

```diff
@@ -38,8 +39,15 @@
 
 namespace Browser {
 
+static String s_search_engine_format = "{}";
```

Comment:
I don't think we need an empty format string here, as there won't be any formatting when it's in this state. Let's just use a null string instead.

```suggestion
static String s_search_engine_format;
```

Reply:
Fixed.